carte_cli/model/carte_table_model.py

```diff
@@ -10,7 +10,8 @@
 
 
 def get_description_text(description: DatabuilderDescription):
-    return description._text
```

Comment:
This change is due to a new minor version of Databuilder that has the description text under the `text` key instead of the former `_text`.